blog-service/2023-05-25-metrics.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Enhanced Heatmap with Custom Dimensions on Y-Axis (Metrics)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - sumo logic
+  - service release notes
+  - open source
+  - metrics
+  - heatmap
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We've enhanced the Heatmap functionality with the addition of custom dimensions on the Y-Axis. This feature enables you to gain deeper insights into your data by visualizing the frequency of occurrences for specific categories within a selected time range.
+
+Using the **Dimensional Heatmap**, you can now not only see the number of errors that occurred within a chosen time range (as with the regular Heatmap) but also effortlessly identify the specific categories, such as cluster/cpu/collector/sourceCategory, where these errors occurred most frequently.
+
+### Configure Dimension Heatmap
+
+To enable the Dimensional Heatmap with Categories on the Y-Axis, follow these steps:
+1. Run a metrics query or logs query with the **transpose** operator.
+<img src={useBaseUrl('img/metrics/custom-heat-map.png')} alt="icon" width="540"/>
+2. Change the Y-Axis Source to **Dimension**
```

Comment:
Change the Y-Axis Source to **Dimension**.

Reply:
What's the change you are suggesting?

Reply:
Period is missing at the end of step 2.

Reply:
```suggestion
2. Change the Y-Axis Source to **Dimension**.
```

Reply:
![Commit suggestion](https://github.com/SumoLogic/sumologic-documentation/assets/116072777/e8caf240-4c27-4c27-8acd-58c3989b8ee1)
That's a helpful suggestion. Please use the Add Suggestion button for these kinds of suggestions. Please see the gif I've attached here.

Reply:
Sure, I will follow this going forward. I was lost finding that button, Thanks for this.